source/getting-started.txt

```diff
@@ -0,0 +1,63 @@
+.. _app-services-get-started:
+
+================
+Getting Started
+================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Template Apps </template-apps>
+   Tutorials </tutorials>
+
+The resources on this page are designed to help you explore Atlas App 
+Services. Visit the Template Apps page to bootstrap an App Services app. 
```

Comment:
I think we could provide more guidance here about what you get with each path. Maybe something like:

> To explore working codebases and apps that showcase App Services features, dive into a Template App. For a more guided experience in learning how to develop with App Services and Realm, check out a Tutorial.